aiocache/backends/redis.py

```diff
@@ -38,41 +37,11 @@ class RedisBackend(BaseCache[str]):
 
     def __init__(
         self,
-        endpoint="127.0.0.1",
-        port=6379,
-        db=0,
-        password=None,
-        pool_min_size=_NOT_SET,
-        pool_max_size=None,
-        create_connection_timeout=None,
+        client,
         **kwargs,
     ):
         super().__init__(**kwargs)
-        if pool_min_size is not _NOT_SET:
-            warnings.warn(
-                "Parameter 'pool_min_size' is deprecated since aiocache 0.12",
-                DeprecationWarning, stacklevel=2
-            )
-
-        self.endpoint = endpoint
-        self.port = int(port)
-        self.db = int(db)
-        self.password = password
-        # TODO: Remove int() call some time after adding type annotations.
-        self.pool_max_size = None if pool_max_size is None else int(pool_max_size)
-        self.create_connection_timeout = (
-            float(create_connection_timeout) if create_connection_timeout else None
-        )
-
-        # NOTE: decoding can't be controlled on API level after switching to
-        # redis, we need to disable decoding on global/connection level
-        # (decode_responses=False), because some of the values are saved as
-        # bytes directly, like pickle serialized values, which may raise an
-        # exception when decoded with 'utf-8'.
```

Comment:
I think this is important. Can we inspect the client and raise an exception if it is not configured correctly?